zerver/middleware.py

```diff
@@ -96,7 +96,7 @@ def write_log_line(log_data, path, method, remote_ip, email, client_name,
         # Remove non-ascii chars from path (there should be none, if there are it's
         # because someone manually entered a nonexistant path), as UTF-8 chars make
         # statsd sad when it sends the key name over the socket
-        statsd_path = statsd_path.encode('ascii', errors='ignore').decode("ascii")
+        statsd_path = statsd_path.encode('ascii', errors='ignore').decode("ascii") # type: ignore # https://github.com/python/mypy/issues/1540
```

Comment:
Cool!  I think when this happens, we should take the opportunity to upgrade our requirements.txt to the latest version of mypy to get the fix and be able to remove the `type: ignore` comment.  Sometimes the new version of mypy will introduce a few new issues that we'll need to track down (or and possibly do new excludes), but I think overall it's net productive to take the upgrades whenever there's benefit to doing so, since the mypy folks are actively fixing issues we haven't run into yet, as well :)